### PR TITLE
Kids seed data

### PR DESCRIPTION
**Pull request by TheTempas**
1-3 kids generate from Faker and assign to a parent, database test verifies!

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -68,14 +68,42 @@
...
     last_name: "of #{Faker::Lovecraft.location}",
     email: "#{Faker::Lovecraft.word}#{count * rand(7) + 1}@campers.com",
     password: "password",
     role: "parent"
+    })
+
+    parent = Parent.create!({
+      user_id: user.id,
+      address: "123 Street",
+      phone_number: "(514) 000-1111"
```
Comment (DerangedMind) [inline db/seeds.rb]: Set this to a simple string of numbers. I believe Rails has a method like to_phone that will automatically convert this into a pretty phone string.

File: db/seeds.rb [hunk 2]
```diff
@@ -68,14 +68,42 @@
...
+      last_name: "of #{Faker::HarryPotter.location}",
+      birthdate: Faker::Date.birthday(5, 14),
+      sin: Faker::Number.number(9),
+      medicare: Faker::Number.number(12)
+    })
+  KidsParent.create!({
+    user_id: user.id,
+    kid_id: kid.id,
+    parent_status: "Status"
   })
```
Comment (DerangedMind) [inline db/seeds.rb]: This block should be indented once more.

Comment (DerangedMind) [inline db/seeds.rb]: As well as random times loop.

File: db/seeds.rb [hunk 3]
```diff
@@ -68,14 +68,42 @@
...
 end
 
+Parent.destroy_all
 puts "Creating Parents..."
 
+Kid.destroy_all
+puts "Creating Kids..."
```
Comment (DerangedMind) [inline db/seeds.rb]: Instead, maybe do `puts "Creating Parents and Kids..."` as it's more explanatory.

**Comment by Garo-A** [comment]
awesome to see Kids-Parents seed! Will be able to really use this during my part tomorrow :)

**Pull request merged**